lxd/instance_file.go

```diff
@@ -477,17 +476,16 @@ func instanceFilePost(s *state.State, inst instance.Instance, path string, r *ht
 		}
 
 		if !exists {
-			// Set file permissions.
```

Comment:
Why did the comment get removed?

Reply:
I removed the comments because they are poor; they tell me exactly what the code does and nothing more. I went ahead and re-added them to reduce noise in the diff.